src/bucket/BucketManagerImpl.cpp

```diff
@@ -184,7 +184,14 @@ BucketManagerImpl::adoptFileAsBucket(std::string const& filename,
         {
             std::string err("Failed to rename bucket :");
             err += strerror(errno);
-            throw std::runtime_error(err);
+            // there is a race condition with external systems (windows)
```

Comment:
If this is only known to occur on Windows, then I think this should be a conditional compile for Windows.

Reply:
I am not sure it's a Windows only problem: I run into it all the time on Windows, that's all I know. I'd rather keep the code the same on both platforms unless this causes problems